python/ray/serialization.py

```diff
@@ -258,6 +258,9 @@ def _deserialize_object(self, data, metadata, object_ref):
                     object_ref.call_site())
             elif error_type == ErrorType.Value("RUNTIME_ENV_SETUP_FAILED"):
                 return RuntimeEnvSetupError()
+            elif error_type == ErrorType.Value(
+                    "CORRESPONDING_PLACEMENT_GROUP_REMOVED"):
```

Comment:
Prefer `PlacementGroupRemoved`

Reply:
Also, can you support two types? TaskPlacementGroupRemoved vs ActorPlacementGroupRemoved

Reply:
updated